python/rapidsmpf/rapidsmpf/integrations/dask/core.py

```diff
@@ -233,8 +222,10 @@ def rmpf_worker_setup(
     """
     ctx = get_worker_context(dask_worker)
     with ctx.lock:
+        ctx.options = options
+
         # Print statistics at worker shutdown.
-        if enable_statistics:
+        if ctx.options.get_or_default("dask_statistics", default_value=False):
```

Comment:
Just to check my understanding: It looks like we need to specify the default value whenever we try to extract a config value, and so the default for the same option can diverge in different parts of the code. There is no central place where user-defined options are validated - Is that correct?

Reply:
Yes, the philosophy is that the component using the option knows how to parse it.